frontend/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    jest: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:prettier/recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: "module",
+  },
+  plugins: ["react", "@typescript-eslint"],
+  rules: {},
```

Comment:
no rules at the moment ?

Reply:
I added two rules. Maybe you suggest some rules. :)

Reply:
I was thinking at some of them.. Maybe worth checking with the others :) @radubrehar 
```
'react/jsx-wrap-multilines': ['error', {
       arrow: true,
       assignment: true,
       declaration: true,
       return: true
     }],
'react/no-unused-state': 'error',
'react/no-will-update-set-state': 'error',
'react/require-render-return': 'error',
'react/self-closing-comp': 'error',
'react/no-multi-comp': ['error', { ignoreStateless: true }],
'react/jsx-no-duplicate-props': ['error', { ignoreCase: true }],
'react/jsx-curly-spacing': ['error', 'never', { allowMultiline: true }],
 'react/jsx-equals-spacing': ['error', 'never']
```

More of them can be found **[here](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules)**

Reply:
There are lots are rules @alincustura already added - but they are in the `extends` clause
```
extends: [
    "eslint:recommended",
    "plugin:prettier/recommended",
    "plugin:react/recommended",
    "plugin:@typescript-eslint/eslint-recommended",
  ],
```
All those plugins/configs contain their own rules.
So I would not sweat over adding too many extra rules besides what "recommended" plugins already bring to the game

Reply:
Great !